react-function-components/src/index.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
```

Comment:
Here you need to specify `client` for `ReactDOM`
```JAVASCRIPT
import ReactDOM from 'react-dom/client';
```

Reply:
<img width="495" alt="Screen Shot 2022-08-31 at 3 20 41 PM" src="https://user-images.githubusercontent.com/97363006/187795667-badff501-e8bd-4b08-8b98-79e7263cb840.png">